tests/features/site_tags.feature

```diff
@@ -20,7 +20,7 @@ Feature: Tagging organizational sites
     When I run "terminus site tags list --site=[[test_site_name]] --org=[[organization_name]] --format=json"
     Then I should get:
     """
-    {"tags":["testtag"]}
+    ["testtag"]
```

Comment:
In lieu of a `Then I should get a JSON model with ...` step, I liked the old test a little better that asserts the whole structure of the JSON object returned. /shrug

We can add better JSON step matcher later that round-trips the output through JSON parsing to a PHP object that can be inspected.